ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -121,6 +135,63 @@ public class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
         return false
     }
 
+    func handleCapacitorHttpRequest(_ urlSchemeTask: WKURLSchemeTask, _ localUrl: URL, _ isHttpsRequest: Bool) {
+        var urlRequest = urlSchemeTask.request
+        let url = urlRequest.url!
```

Comment:
Probably better to `guard let url = urlRequest.url else { return }` than to force unwrap. The only time it should be considered okay to force unwrap is if you can guarantee without a doubt that the optional is present, either because you control the data flow yourself, or you've otherwise checked its presence in a way that didn't lead to the safe unwrapping. It probably shouldn't be considered okay to crash the program here if the url isn't present.

Reply:
I didn't comment on that because we already had a force unwrap in a very similar code here https://github.com/ionic-team/capacitor/blob/main/ios/Capacitor/Capacitor/WebViewAssetHandler.swift#L25
If it didn't crash there it would crash in the new code neither.

But yeah, we supposedly had rules in swiftlint to not allow force unwraps, but doesn't seem to be working.